benchmarks/benchmarks/optimize.py

```diff
@@ -209,7 +209,8 @@ def bench_run(self, x0, methods=None, **minimizer_kwargs):
         if methods is None:
             methods = ["COBYLA", 'Powell',
                        'L-BFGS-B', 'BFGS', 'CG', 'TNC', 'SLSQP',
-                       "Newton-CG", 'dogleg', 'trust-ncg', 'trust-region-exact']
+                       "Newton-CG", 'dogleg', 'trust-ncg', 'trust-trlib',
```

Comment:
I think the name 'trust-trlib' for the method is not very good. Can you think of a better one? I would go for something like 'trust-lanczos'....